.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,259 @@
+
+# Table of Contents
+
+- [Reporting Bugs](#reporting-bugs)
+  * [Upstream bugs](#upstream-bugs)
+- [Contributing patches and new features](#contributing-patches-and-new-features)
+  * [Branches](#branches)
+  * [Picking an open issue](#picking-an-open-issue)
+  * [Contributing to the project](#contributing-to-the-project)
+- [Considerations when writing sniffs](#considerations-when-writing-sniffs)
+  * [Unit Testing](#unit-testing)
+    + [Pre-requisites](#pre-requisites)
+    + [Composer setup](#composer-setup)
+    + [Other setups](#other-setups)
+      - [Method 1](#method-1)
+      - [Method 2](#method-2)
+  * [Writing and running unit tests](#writing-and-running-unit-tests)
+  * [Unit Testing conventions](#unit-testing-conventions)
+    + [Public properties](#public-properties)
+    + [File organization and naming](#file-organization-and-naming)
+  * [Code Standards for this project](#code-standards-for-this-project)
+
+# Reporting Bugs
+
+When reporting a bug in an existing sniff it's good to differentiate on whether the sniff is reporting something which the sniff shouldn't report on (a _false positive_), or whether the sniff isn't reporting on a certain thing that it should report on (a _false negative_).
+
+In the case you found something that a sniff shouldn't report on, you can check where it's coming from using the `-s` flag with the `phpcs` command.
+
+In case the sniff is not reporting on a certain thing it should, you won't be able to check it using the `-s` flag, but **in both** cases it's **mandatory** to provide code samples so that the issue could be easily remedied.
+
+## Upstream bugs
+
+In the case the sniff error code doesn't starts with `WPThemeReview`, and instead it starts with `WordPress`, `PHPCompatibility`, or something else, that means that it is and 'upstream' bug coming from either `WPCS`, `PHPCompatibility` or `PHPCS`. You can report the bug here, but the chances are high that you'll be asked to report it in the correct repository instead.
+
+# Contributing patches and new features
+
+## Branches
+
+Ongoing development will be done in the `develop` branch with merges done into `master` once considered stable.
+
+## Picking an open issue
+
+If you want to pick an open issue, please mention that in the issue, and it will be assigned to you. This will prevent the possible double work by different people on the same issue.
+
+## Contributing to the project
+
+If you want to contribute to this project, fork the repo and open a pull request to the `develop` branch. If you have push access to this repo, you can create a feature branch and then open an intra-repo PR from that branch to the `develop`.
+
+# Considerations when writing sniffs
+
+## Unit Testing
+
+All PRs which affect sniffs, whether bug fixes to existing sniffs, or the addition of a new sniff, should be accompanied by unit tests.
+
+### Pre-requisites
+
+* PHP 5.4 or higher
```

Comment:
I think we can remove the pre-requisites, except for the PHP requirement.

Everything else will normally be installed via Composer, so is not a prerequisite.